miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -428,12 +438,39 @@ end
 #! tag is the tag to be included in the note.
 #! RECIPIENT is the recipient of the note.
 #! ptr is the pointer to the memory address at which the note is stored.
-export.create_note
+export.create_offchain_note
     # authenticate that the procedure invocation originates from the account context
     exec.authenticate_account_origin
     # => [ASSET, tag, RECIPIENT]
 
-    # create the note
+    push.OFFCHAIN_NOTE movdn.6
+    # => [ASSET, tag, note_type, RECIPIENT]
+
+    # create the offchain note
+    exec.tx::create_note
+    # => [ptr, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Firing of note creation events would be a part of a follow-up PR, right?

Reply:
yes, this one is not doing it yet.